### PR TITLE
Game Support: Add GTA SA

### PR DESCRIPTION
**Pull request by felix-schindler**

### CODE REVIEW

**Review by mattmaniak — CHANGES_REQUESTED** [review]
Please run the "generate" script to add the game to games list.

File: src/game-support/gta-sa.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "GTA San Andreas",
+    "Grand Theft Auto SA"
+] -->
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+> [!WARNING]
+> The Rockstar Games Launcher will fail with the `The Rockstar Games Launcher failed to initialize!` error when `DXVK` is turned `off` in your `Bottle Configuration`. `DXVK` can turned `on` to workaround the launcher error.
+
+Crashes during load without error message and is therefore currently unplayable in Whisky.
```
Comment (mattmaniak) [inline src/game-support/gta-sa.md:13]: Does the game crashes itself or the Rockstar Launcher? Same question here.

Comment (felix-schindler) [reply src/game-support/gta-sa.md:13]: The game itself.

The Rockstar Launcher works fine (with DXVK turned on), you can download and install the game and it's requirements. When you launch the game, it immediately crashes.

Comment (mattmaniak) [reply src/game-support/gta-sa.md:13]: Understood. Please specify that it's the game case as it is ambigous a bit now.

**Comment by felix-schindler** [comment]
superseded by #105

**Pull request closed without merging**